cmd/argo/commands/server.go

```diff
@@ -224,7 +213,6 @@ See %s`, help.ArgoServer),
 	command.Flags().StringVar(&configMap, "configmap", common.ConfigMapName, "Name of K8s configmap to retrieve workflow controller configuration")
 	command.Flags().BoolVar(&namespaced, "namespaced", false, "run as namespaced mode")
 	command.Flags().StringVar(&managedNamespace, "managed-namespace", "", "namespace that watches, default to the installation namespace")
-	command.Flags().StringVar(&ssoNamespace, "sso-namespace", "", "namespace that will be used for SSO RBAC. Defaults to installation namespace. Used only in namespaced mode")
```

Comment:
This is a breaking change. I see it was discussed above, but it wasn't mentioned in the changelog or release notes. The PR title is also missing an exclamation point indicating breakage 😕